motionplan/tpSpaceRRT.go

```diff
@@ -1041,3 +993,54 @@ func rectifyTPspacePath(path []node, frame referenceframe.Frame, startPose spati
 	}
 	return correctedPath, nil
 }
+
+func extractTPspacePath(startMap, goalMap map[node]node, pair *nodePair) []node {
+	// need to figure out which of the two nodes is in the start map
+	var startReached, goalReached node
+	if _, ok := startMap[pair.a]; ok {
+		startReached, goalReached = pair.a, pair.b
+	} else {
+		startReached, goalReached = pair.b, pair.a
+	}
+
+	// extract the path to the seed
+	path := make([]node, 0)
+	for startReached != nil {
+		path = append(path, startReached)
+		startReached = startMap[startReached]
+	}
+
+	// reverse the slice
+	for i, j := 0, len(path)-1; i < j; i, j = i+1, j-1 {
+		path[i], path[j] = path[j], path[i]
+	}
+
+	if goalReached != nil {
```

Comment:
I'm not sure when this would ever be useful

Reply:
It's not anymore; will remove.